systemtest/src/test/java/io/strimzi/systemtest/operators/PodSetST.java

```diff
@@ -46,6 +50,16 @@ void testPodSetOnlyReconciliation(ExtensionContext extensionContext) {
         final int replicas = 3;
         final int probeTimeoutSeconds = 6;
 
+        // setup clients
+        KafkaClients clients = new KafkaClientsBuilder()
```

Comment:
I am also thinking about continuous sending messages during update of FG? Do you think it is worth @im-konge @henryZrncik ?

Reply:
The changes would be, as some of brokers will go down (not only mentioning reconciliation),   config producer to `ack=al`l, broker to create topics by default with appropriate `min.insync.replicas` and `replicaiton` factor). Similar workaround  was done in `SecurityST testCaRenewalBreakInMiddle`

But I do think it is worth adding there.

Reply:
I added suggestions into the code base. 👍